sass/true/_assert.scss

```diff
@@ -153,6 +153,31 @@
   @include _true-assert-results($assert, $expected, 'unequal');
 }
 
+// Assert Error
+// ------------
+/// Assert that a mixin or function triggers an `@error`.
+///
+/// @group api-assert-values
+///
+/// @param {string} $message -
+///   Optional message to check against the emitted message.
+///   The emitted message must contain the expected message,
+///   allowing for partial message validation.
+///   This is a case-insensitive check.
+///
+///
+/// @example scss -
+///   @include test('Some mixin does what it is supposed to do') {
+///     @include assert-error('must pass valid params'){
+///       @include some-mixin($bad-params);
+///     }
+///   }
+@mixin assert-error($message: null) {
+  @if (false) {
```

Comment:
I still think it would be useful for manual testing to allow a variable override to be passed in – and setting that variable also seems simpler than search-replace of the entire mixin call. Since a global variable could be passed to all mixins, that could potentially also help resolve @jgerigmeyer's note about testing imported files?

Reply:
I am not entirely opposed to this. However, this introduces some complexities which might send a user on a wild red herring chase. For example, let's say a user wants to manually test an error by passing `true`, there could another unrelated error elsewhere in the file. Users might think that their error code is being hit when in fact it is not.

Regarding the global variable thing... due to the fact that errors are fatal, we must still do one-by-one replacements for error assertions as they must be tested one at a time. There's no way to set a single global variable but only have it affect one instance of a mixin. 

All this said, I'll go ahead and add a second parameter so you can test it and see how it feels. If you want it to stay, I'm all for it.

Reply:
Those are great points, makes sense.